internal/service/federatedsettingsidentityprovider/data_source_federated_settings_identity_provider_test.go

```diff
@@ -36,6 +36,35 @@ func TestAccFedDSFederatedSettingsIdentityProvider_basic(t *testing.T) {
 	})
 }
 
+func TestAccFedDSFederatedSettingsIdentityProvider_oidcBasic(t *testing.T) {
+	acc.SkipTestExtCred(t)
+	var (
+		resourceName        = "data.mongodbatlas_federated_settings_identity_provider.test"
+		federatedSettingsID = os.Getenv("MONGODB_ATLAS_FEDERATION_SETTINGS_ID")
+		idpID               = os.Getenv("MONGODB_ATLAS_FEDERATED_OIDC_IDP_ID")
+	)
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acc.PreCheckFederatedSettings(t) },
+		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccMongoDBAtlasDataSourceFederatedSettingsIdentityProviderConfig(federatedSettingsID, idpID),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckMongoDBAtlasFederatedSettingsIdentityProvidersExists(resourceName),
+
+					resource.TestCheckResourceAttrSet(resourceName, "federation_settings_id"),
```

Comment:
can we also check for the expected value or you think it's not so important?

Reply:
I think that testing that the attribute itself is set it's enough. In the unit tests in model_federated_settings_identity_provider_test.go we are already testing the values.

Reply:
I agree with you we're using unit tests and we shouldn't really "test the other components", but think about an API change that changes a computed value (like it's been happening recently): how would we detect it if not through these tests?

Reply:
Good point! I included some value checks in both OIDC and SAML acceptance test to check most important attributes values